app/models/items/recurring_donation.rb

```diff
@@ -0,0 +1,9 @@
+class RecurringDonation < Item
+
+  belongs_to :account_code
+  belongs_to :customer
+  has_many :donations, foreign_key: :recurring_donation_id
+
+  def one_line_description ; end
```

Comment:
`one_line_description` and `description_for_audit_txn` should be filled in for this subclass.  take a look at how they're defined in `Donation` and `Voucher` subclasses for guidance.

Reply:
changed